modules/ROOT/pages/conf-examples/office-integration.adoc

```diff
@@ -0,0 +1,124 @@
+
+You need a Linux server with docker and docker-compose installed.
+
+Then you need to set up the following domains pointing to your server:
+
+* ocis.* for serving Infinite Scale
+* collabora.* for serving Collabora
+* onlyoffice.* for serving OnlyOffice
+* wopiserver.* for serving the WOPI server
+* traefik.* for serving the Traefik dashboard
+
+The Traefik stack can also be run locally. One downside is, that Traefik can not obtain valid SSL certificates and therefore will create self-signed ones. This means that your browser will show security warnings. Another downside is, that you cannot point DNS entries to your localhost. So you have to add static host entries to your system. On Linux and macOS you can add them to your `/etc/hosts` file like this:
+
+[source,plaintext]
+----
+127.0.0.1 ocis.owncloud.test
+127.0.0.1 traefik.owncloud.test
+127.0.0.1 collabora.owncloud.test
+127.0.0.1 onlyoffice.owncloud.test
+127.0.0.1 wopiserver.owncloud.test
+----
+
+For the clients, make sure that the addresses can be resolved to the server(s) running this setup.
+
+== Installation
+
+Installing the Docker stacks only requires a few steps. Take care to use the relevant version. Select a base folder of choice for the target location. Note that the examples use the latest stable version: {compose_tab_2}.
+
+. Get the deployment example:
++
+--
+Follow the xref:deployment/wopi/wopi.html#wopi-coniguration-examples[WOPI Coniguration Examples,window=_blank] to download the example files.
+
+After all resources have been downloaded, change into the directory:
+
+[source,bash]
+----
+cd ocis_wopi
+----
+--
+
+== Configuration
+
+The configuration uses a docker-compose file with an environment variable file for ease of handling. This docker-compose file defines which additional containers are needed and how the containers are configured.
+
+[start=2]
+. Firstly, the xref:conf-examples/ext-files/list_file_1.adoc[environment definition file,window=_blank] needs to be adapted. This file defines many basic settings in one location. Open it in a text editor and adjust it according to your needs.
++
+--
+For some hints on what to take into consideration, check the notes listed here:
+
+* If you are installing Infinite Scale on a server and Traefik will obtain valid certificates for you, remove `INSECURE=true` or set it to `false`.
+
+* If you want to use the Traefik dashboard, set `TRAEFIK_DASHBOARD` to `true` (default is `false` and therefore not active). If you activate it, you must set a domain for the Traefik dashboard in `TRAEFIK_DOMAIN=`, for example `TRAEFIK_DOMAIN=traefik.owncloud.test`.
+
+* The Traefik dashboard is secured by Basic Auth. Default credentials are the user `admin` with the password `admin`. To set your own credentials, generate a htpasswd (e.g. by using an https://htpasswdgenerator.de/[online tool] or a CLI tool).
+
+* Traefik will issue certificates with LetsEncrypt. Therefore you must set an email address in `TRAEFIK_ACME_MAIL=`.
+
```

Comment:
Please remove the part about the Traefik Dashboard. Not applicable for this documentation.

Reply:
We are not the master/creator of the compose file which is referenced by this text. The compose file was created by Willy. We take what he provides. This is now "locked" in the v2.0.0. branch. If a change is required, one with the right knowledge can do it in the master branch of the ocis repo. When the next release like v2.1 is published, we can then change the description accordingly. You can see the whole picture more easily when looking into staging where you can see a rendered build...